README.md

```diff
@@ -1,16 +1,21 @@
-# Title
+# Introduction to Active Record Associations
 
-## Learning Goals
+While you might be in the early days of writing code and learning how to program, 
+you have some  assets on your side: a positive attitude, a community you can ask 
+for help, and  a history of solving problems on your own. In this module, we're going 
+to give you  some language and a framework for the strategies that you should embrace 
+as you start solving problems in code.
 
--SWBAT 1
--SWBAT 2
+One of the most important skillsets for web development is the ability to think logically and to break down down problems into smaller, component pieces. This skillset is required when setting up your database. Data is the foundation of almost all web applications.
 
-## Introduction
+Active Record is the interface that Rails gives you to link the database to your application. It lets you structure your data models in a logical and nearly plain-English way. 
```

Comment:
Students haven't gotten to Rails yet. It could be mentioned potentially, but as something coming up that will use Active Record.  Talking about Rails might also be a good thing to save for the bookend of this brick, as well.